applications/RANSApplication/tests/test_RANSApplication.py

```diff
@@ -14,8 +14,12 @@
 # Import the tests o test_classes to create the suites
 import run_cpp_unit_tests
 
+# process test_classes
 from custom_process_tests import CustomProcessTest
 
+# flow solver test_classes
+from incompressible_potential_flow_solver_formulation_tests import IncompressiblePotentialFlowSolverFormulationTest
+
 def AssembleTestSuites():
     ''' Populates the test suites to run.
 
```

Comment:
No small test?

Reply:
I am thinking to add few k-omega-sst tests as small tests in coming PR, so RANSApplication won't take in small test execution time. All the other tests are added to nightly.
These k-omega-sst tests should cover most of the aspects of RANSApplication.